lua/indent_blankline/init.lua

```diff
@@ -143,6 +145,11 @@ local refresh = function()
     }
 
     local max_indent_level = v("indent_blankline_indent_level")
+    local max_indent_increase = v("indent_blankline_max_indent_increase")
+    if max_indent_increase <= 0 then
+        max_indent_increase = max_indent_level
+    end
+    local last_normal_indent = -1
```

Comment:
Can you rename this to `previous_indent` and move it to line 288, above `next_indent`

Also, just start with `0`, then you can remove the `previous_indent ~= -1` checks

Reply:
> just start with `0`

This will cause problem if the first line is indented.

Reply:
What problem exactly? Maybe I'm missing it, but I don't see anything wrong

Reply:
If the whole viewport is indented, all lines are treated as abnormal lines.

Reply:
:+1: yeah you are right